maas/plugins/galera_check.py

```diff
@@ -74,8 +74,8 @@ def print_metrics(replica_status):
            replica_status['wsrep_commit_window'], 'sequence_delta')
     metric('wsrep_cluster_size', 'int64',
            replica_status['wsrep_cluster_size'], 'nodes')
-    metric('queries_per_second', 'int64',
-           replica_status['Queries'], 'qps')
+    metric('queries', 'int64',
```

Comment:
I'd say we should be true to what we would do for incompat changes. 'Deprecate then Delete' cycle. So provide both now, and document the change for consumers. file a bug for Newton, remove the old one then.